### PR TITLE
PLAT-414: call self.Destroy from TestWallet

### PR DESCRIPTION
**Pull request by ElSacramento**
add call self.Destroy for TestWallet contract

### CODE REVIEW

File: ledger-core/virtual/integration/method_api_test.go [hunk 1]
```diff
@@ -102,5 +102,15 @@ func TestVirtual_Method_API(t *testing.T) {
 			assert.NotEmpty(t, response.TraceID)
 			assert.Equal(t, uint(1000000000), response.Amount)
 		}
+		// { // Delete request
```
Comment (ElSacramento) [inline ledger-core/virtual/integration/method_api_test.go:105]: creates panic(not implemented) now

Comment (ElSacramento) [reply ledger-core/virtual/integration/method_api_test.go:105]:
```
2020-07-20T15:39:39.848576000+03:00 ERR 0xc00029c000[116] 026 @ 018: panic current=(*SMExecute).stepSaveNewObject next=<nil> timing=1ns/1ns payload=*execute.SMExecute tracer=6c44fbea7f4bdbf512637faa35a0e685 errorMsg=execution: not implemented
Stack trace:
panic(0x1d0cc60, 0xc000151540)
	/usr/local/Cellar/go/1.14/libexec/src/runtime/panic.go:973 +0x396
github.com/insolar/assured-ledger/ledger-core/virtual/execute.(*SMExecute).stepSaveNewObject(0xc000655100, 0x1fd11e0, 0xc000148de0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/virtual/execute/execute.go:770 +0xa1e
github.com/insolar/assured-ledger/ledger-core/conveyor/smachine.(*executionContext).executeNextStep(0xc000148de0, 0xc000148de0, 0x1, 0xffffffff, 0x0, 0x0, 0xc0001534c0, 0x0, 0x0, 0x0, ...)
	/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/smachine/context_exec.go:128 +0x108

2020-07-20T15:39:39.850032000+03:00 ERR execution: not implemented caller=virtual/integration/convlog/logger.go:118 details="{panic {{<nil> <nil> <nil> 0} {0 }} <nil> 1ns 1ns}" err=panic isComparable=false stack="github.com/insolar/assured-ledger/ledger-core/conveyor/smachine.recoverSlotPanicAsUpdate(0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/smachine/state_update_type.go:63 +0x226\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/smachine.(*contextTemplate).discardAndUpdate(...)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/smachine/context_basic.go:93\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/smachine.(*executionContext).executeNextStep.func1(0xc000148de0, 0xc00014bbd0)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/smachine/context_exec.go:123 +0x11f\npanic(0x1d0cc60, 0xc000151540)\n\t/usr/local/Cellar/go/1.14/libexec/src/runtime/panic.go:973 +0x396\ngithub.com/insolar/assured-ledger/ledger-core/virtual/execute.(*SMExecute).stepSaveNewObject(0xc000655100, 0x1fd11e0, 0xc000148de0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/virtual/execute/execute.go:770 +0xa1e\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/smachine.(*executionContext).executeNextStep(0xc000148de0, 0xc000148de0, 0x1, 0xffffffff, 0x0, 0x0, 0xc0001534c0, 0x0, 0x0, 0x0, ...)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/smachine/context_exec.go:128 +0x108\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/smachine.(*SlotMachine)._executeSlot.func1(0x1fba900, 0xc00015cb28)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/smachine/slot_machine_execute.go:232 +0x2b2\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/sworker.(*SimpleSlotWorker).DetachableCall(0xc00015cb00, 0xc00014ba40, 0x2868f00)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/sworker/worker_simple.go:118 +0x44\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/smachine.(*SlotMachine)._executeSlot(0xc00029c000, 0xc000580100, 0x10, 0x1fb57a0, 0xc00015cb00, 0x2711, 0x0, 0x1)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/smachine/slot_machine_execute.go:213 +0x235\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/smachine.(*SlotMachine).executeWorkingSlots(0xc00029c000, 0x74, 0x1fb57a0, 0xc00015cb00)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/smachine/slot_machine_execute.go:193 +0x1b3\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/smachine.(*SlotMachine).ScanOnce(0xc00029c000, 0x0, 0x1fb57a0, 0xc00015cb00, 0xc00021d2a0, 0x100ec18, 0x20, 0x1c6dbc0)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/smachine/slot_machine_execute.go:120 +0x45b\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/smachine.(*SlotMachine).ScanNested.func1(0x1fb57a0, 0xc00015cb00)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/smachine/slot_machine_execute.go:35 +0x5c\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/sworker.(*AttachableSimpleSlotWorker).AttachAsNested(0xc0001a1074, 0xc00029c000, 0x1fba900, 0xc00015cae8, 0x2710, 0xc000148d20, 0x203000)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/sworker/worker_simple.go:45 +0x145\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/smachine.(*SlotMachine).ScanNested(0xc00029c000, 0x1fd11e0, 0xc000148cf0, 0x27100014b900, 0x1f98de0, 0xc0001a1074, 0x1d42320, 0xc00061bf80, 0x0, 0x40)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/smachine/slot_machine_execute.go:34 +0x11f\ngithub.com/insolar/assured-ledger/ledger-core/conveyor.(*PulseSlotMachine).stepPresentLoop(0xc0001811d0, 0x1fd11e0, 0xc000148cf0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/sm_pulse_slot.go:215 +0x8a\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/smachine.(*executionContext).executeNextStep(0xc000148cf0, 0xc000148cf0, 0x1, 0x0, 0xc00045e000, 0xc00021d7a8, 0x14c58e8, 0xc000500000, 0xc00061bf80, 0x0, ...)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/smachine/context_exec.go:128 +0x108\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/smachine.(*SlotMachine)._executeSlot.func1(0x1fba900, 0xc00015cae8)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/smachine/slot_machine_execute.go:232 +0x2b2\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/sworker.(*SimpleSlotWorker).DetachableCall(0xc00015cac0, 0xc00014b900, 0x2868f00)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/sworker/worker_simple.go:118 +0x44\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/smachine.(*SlotMachine)._executeSlot(0xc00045e000, 0xc000500100, 0x2, 0x1fb57a0, 0xc00015cac0, 0x1389, 0x0, 0x0)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/smachine/slot_machine_execute.go:213 +0x235\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/smachine.(*SlotMachine).executeWorkingSlots(0xc00045e000, 0x46, 0x1fb57a0, 0xc00015cac0)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/smachine/slot_machine_execute.go:193 +0x1b3\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/smachine.(*SlotMachine).ScanOnce(0xc00045e000, 0x0, 0x1fb57a0, 0xc00015cac0, 0xc00021de40, 0x100ec18, 0x40, 0x1d19760)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/smachine/slot_machine_execute.go:120 +0x45b\ngithub.com/insolar/assured-ledger/ledger-core/conveyor.(*PulseConveyor).runWorker.func2(0x1fb57a0, 0xc00015cac0)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/conveyor.go:584 +0x60\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/sworker.(*AttachableSimpleSlotWorker).AttachTo(0xc0000acf80, 0xc00045e000, 0xc000294500, 0xffffffff, 0xc000151420, 0x0, 0x0)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/sworker/worker_simple.go:61 +0xe1\ngithub.com/insolar/assured-ledger/ledger-core/conveyor.(*PulseConveyor).runWorker(0xc00043b180, 0x0, 0xc0000c4420, 0xc000424d50, 0xc000424be0)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/conveyor.go:583 +0x1bd\ncreated by github.com/insolar/assured-ledger/ledger-core/conveyor.(*PulseConveyor).StartWorkerExt\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/conveyor.go:553 +0xe3" testname=TestVirtual_Method_API
    TestVirtual_Method_API: console_testing.go:83: 2020-07-20T15:39:39.850032000+03:00 ERR execution: not implemented caller=virtual/integration/convlog/logger.go:118 details="{panic {{<nil> <nil> <nil> 0} {0 }} <nil> 1ns 1ns}" err=panic isComparable=false stack="github.com/insolar/assured-ledger/ledger-core/conveyor/smachine.recoverSlotPanicAsUpdate(0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/smachine/state_update_type.go:63 +0x226\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/smachine.(*contextTemplate).discardAndUpdate(...)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/smachine/context_basic.go:93\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/smachine.(*executionContext).executeNextStep.func1(0xc000148de0, 0xc00014bbd0)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/smachine/context_exec.go:123 +0x11f\npanic(0x1d0cc60, 0xc000151540)\n\t/usr/local/Cellar/go/1.14/libexec/src/runtime/panic.go:973 +0x396\ngithub.com/insolar/assured-ledger/ledger-core/virtual/execute.(*SMExecute).stepSaveNewObject(0xc000655100, 0x1fd11e0, 0xc000148de0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/virtual/execute/execute.go:770 +0xa1e\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/smachine.(*executionContext).executeNextStep(0xc000148de0, 0xc000148de0, 0x1, 0xffffffff, 0x0, 0x0, 0xc0001534c0, 0x0, 0x0, 0x0, ...)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/smachine/context_exec.go:128 +0x108\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/smachine.(*SlotMachine)._executeSlot.func1(0x1fba900, 0xc00015cb28)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/smachine/slot_machine_execute.go:232 +0x2b2\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/sworker.(*SimpleSlotWorker).DetachableCall(0xc00015cb00, 0xc00014ba40, 0x2868f00)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/sworker/worker_simple.go:118 +0x44\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/smachine.(*SlotMachine)._executeSlot(0xc00029c000, 0xc000580100, 0x10, 0x1fb57a0, 0xc00015cb00, 0x2711, 0x0, 0x1)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/smachine/slot_machine_execute.go:213 +0x235\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/smachine.(*SlotMachine).executeWorkingSlots(0xc00029c000, 0x74, 0x1fb57a0, 0xc00015cb00)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/smachine/slot_machine_execute.go:193 +0x1b3\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/smachine.(*SlotMachine).ScanOnce(0xc00029c000, 0x0, 0x1fb57a0, 0xc00015cb00, 0xc00021d2a0, 0x100ec18, 0x20, 0x1c6dbc0)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/smachine/slot_machine_execute.go:120 +0x45b\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/smachine.(*SlotMachine).ScanNested.func1(0x1fb57a0, 0xc00015cb00)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/smachine/slot_machine_execute.go:35 +0x5c\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/sworker.(*AttachableSimpleSlotWorker).AttachAsNested(0xc0001a1074, 0xc00029c000, 0x1fba900, 0xc00015cae8, 0x2710, 0xc000148d20, 0x203000)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/sworker/worker_simple.go:45 +0x145\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/smachine.(*SlotMachine).ScanNested(0xc00029c000, 0x1fd11e0, 0xc000148cf0, 0x27100014b900, 0x1f98de0, 0xc0001a1074, 0x1d42320, 0xc00061bf80, 0x0, 0x40)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/smachine/slot_machine_execute.go:34 +0x11f\ngithub.com/insolar/assured-ledger/ledger-core/conveyor.(*PulseSlotMachine).stepPresentLoop(0xc0001811d0, 0x1fd11e0, 0xc000148cf0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/sm_pulse_slot.go:215 +0x8a\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/smachine.(*executionContext).executeNextStep(0xc000148cf0, 0xc000148cf0, 0x1, 0x0, 0xc00045e000, 0xc00021d7a8, 0x14c58e8, 0xc000500000, 0xc00061bf80, 0x0, ...)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/smachine/context_exec.go:128 +0x108\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/smachine.(*SlotMachine)._executeSlot.func1(0x1fba900, 0xc00015cae8)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/smachine/slot_machine_execute.go:232 +0x2b2\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/sworker.(*SimpleSlotWorker).DetachableCall(0xc00015cac0, 0xc00014b900, 0x2868f00)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/sworker/worker_simple.go:118 +0x44\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/smachine.(*SlotMachine)._executeSlot(0xc00045e000, 0xc000500100, 0x2, 0x1fb57a0, 0xc00015cac0, 0x1389, 0x0, 0x0)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/smachine/slot_machine_execute.go:213 +0x235\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/smachine.(*SlotMachine).executeWorkingSlots(0xc00045e000, 0x46, 0x1fb57a0, 0xc00015cac0)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/smachine/slot_machine_execute.go:193 +0x1b3\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/smachine.(*SlotMachine).ScanOnce(0xc00045e000, 0x0, 0x1fb57a0, 0xc00015cac0, 0xc00021de40, 0x100ec18, 0x40, 0x1d19760)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/smachine/slot_machine_execute.go:120 +0x45b\ngithub.com/insolar/assured-ledger/ledger-core/conveyor.(*PulseConveyor).runWorker.func2(0x1fb57a0, 0xc00015cac0)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/conveyor.go:584 +0x60\ngithub.com/insolar/assured-ledger/ledger-core/conveyor/sworker.(*AttachableSimpleSlotWorker).AttachTo(0xc0000acf80, 0xc00045e000, 0xc000294500, 0xffffffff, 0xc000151420, 0x0, 0x0)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/sworker/worker_simple.go:61 +0xe1\ngithub.com/insolar/assured-ledger/ledger-core/conveyor.(*PulseConveyor).runWorker(0xc00043b180, 0x0, 0xc0000c4420, 0xc000424d50, 0xc000424be0)\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/conveyor.go:583 +0x1bd\ncreated by github.com/insolar/assured-ledger/ledger-core/conveyor.(*PulseConveyor).StartWorkerExt\n\t/Users/elsacramento/go/src/github.com/insolar/assured-ledger/ledger-core/conveyor/conveyor.go:553 +0xe3" testname=TestVirtual_Method_API
```

Comment (ivanshibitov) [reply ledger-core/virtual/integration/method_api_test.go:105]: May be mark when it's going to be fixed?

Comment (aalekseev-ru) [reply ledger-core/virtual/integration/method_api_test.go:105]: because it goes to stepSaveNewObject -> case requestresult.SideEffectDeactivate:

Comment (ElSacramento) [reply ledger-core/virtual/integration/method_api_test.go]: done

**Pull request merged**